libs/application/templates/car-recycling/src/fields/VehiclesOverview/VehiclesOverview.tsx

```diff
@@ -132,14 +162,11 @@ const VehiclesOverview: FC<FieldBaseProps> = ({
     }
   }
 
-  function onCancel(vehicle: VehicleDto): void {
+  const onCancel = (vehicle: VehicleDto): void => {
     // Check if the vehicle has been selcted and submitted
     if (vehicle.selectedForRecycling) {
```

Comment:
Isn't it only selected vehicles that have this onCancel method? Then you don't need this if statement since it would always be true